nautobot/wireless/models.py

```diff
@@ -121,20 +121,25 @@ class RadioProfile(PrimaryModel):
     frequency = models.CharField(
         max_length=CHARFIELD_MAX_LENGTH,
         choices=RadioProfileFrequencyChoices,
+        blank=True,
     )
-    tx_power_min = models.IntegerField()
-    tx_power_max = models.IntegerField()
+    tx_power_min = models.IntegerField(blank=True, null=True)
+    tx_power_max = models.IntegerField(blank=True, null=True)
     channel_width = JSONArrayField(
-        base_field=models.IntegerField(choices=RadioProfileChannelWidthChoices), default=list
+        base_field=models.IntegerField(choices=RadioProfileChannelWidthChoices),
+        default=list,
+        blank=True,
+    )
+    allowed_channel_list = JSONArrayField(
+        base_field=models.IntegerField(blank=True, null=True), default=list, blank=True, null=True
     )
```

Comment:
Wonder if either of these should use the `PositiveRangeNumberTextField` being added in #6176?

Reply:
Channel width was requested to be multiple choices. We don’t have a multiple static choice field, but implementing as part of the JSONArray worked well.

allowed_channel_list could probably be the RangeNumber though, I will double check with Merek.

Reply:
There may be a benefit for the 2.4GHz channels as these can go from 1-14, but the 6GHz channels are not consecutive.

Reply:
We are going to stick with a JSONArrayField.